openmc/tallies.py

```diff
@@ -2748,7 +2748,7 @@ def get_slice(self, scores=[], filters=[], filter_bins=[], nuclides=[]):
                         bin_indices.append(bin_index)
                         num_bins += 1
 
-                find_filter.bins = set(find_filter.bins[bin_indices])
+                find_filter.bins = np.unique(find_filter.bins[bin_indices])
```

Comment:
This function was added in NumPy 1.9. We should probably change the install_requires line in setup.py to say `numpy>=1.9`.